### ISSUE TITLE
Publish wheels for linux on pypi

### ISSUE THREAD

**Issue by arthurlogilab** [xattr/xattr#85]
https://pypi.org/project/xattr/#files indicates only wheels for macosx, any chance to get some wheels for linux ? 

I'm getting compile errors in our CI for trying to install xattr without python-dev 

```
    Running setup.py install for xattr: finished with status 'error'
    ERROR: Command errored out with exit status 1:
     command: /.tox/py27/bin/python -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-tq12CM/xattr/setup.py'"'"'; __file__='"'"'/tmp/pip-install-tq12CM/xattr/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record /tmp/pip-record-jrfrxk/install-record.txt --single-version-externally-managed --compile --install-headers /.tox/py27/include/site/python2.7/xattr
         cwd: /tmp/pip-install-tq12CM/xattr/
    Complete output (24 lines):
    running install
    running build
    running build_py
    creating build
    creating build/lib.linux-x86_64-2.7
    creating build/lib.linux-x86_64-2.7/xattr
    copying xattr/__init__.py -> build/lib.linux-x86_64-2.7/xattr
    copying xattr/lib.py -> build/lib.linux-x86_64-2.7/xattr
    copying xattr/pyxattr_compat.py -> build/lib.linux-x86_64-2.7/xattr
    copying xattr/compat.py -> build/lib.linux-x86_64-2.7/xattr
    copying xattr/lib_build.py -> build/lib.linux-x86_64-2.7/xattr
    copying xattr/tool.py -> build/lib.linux-x86_64-2.7/xattr
    running build_ext
    generating cffi module 'build/temp.linux-x86_64-2.7/_lib.c'
    creating build/temp.linux-x86_64-2.7
    building '_lib' extension
    creating build/temp.linux-x86_64-2.7/build
    creating build/temp.linux-x86_64-2.7/build/temp.linux-x86_64-2.7
    x86_64-linux-gnu-gcc -pthread -DNDEBUG -g -fwrapv -O2 -Wall -Wstrict-prototypes -fno-strict-aliasing -Wdate-time -D_FORTIFY_SOURCE=2 -g -fdebug-prefix-map=/build/python2.7-IbFBHb/python2.7-2.7.16=. -fstack-protector-strong -Wformat -Werror=format-security -fPIC -I/usr/include/python2.7 -c build/temp.linux-x86_64-2.7/_lib.c -o build/temp.linux-x86_64-2.7/build/temp.linux-x86_64-2.7/_lib.o
    build/temp.linux-x86_64-2.7/_lib.c:22:12: fatal error: pyconfig.h: No such file or directory
     #  include <pyconfig.h>
                ^~~~~~~~~~~~
    compilation terminated.
    error: command 'x86_64-linux-gnu-gcc' failed with exit status 1
```

**Comment by etrepum** [comment]
Building wheels for Linux is still rather complicated, if someone were to submit a PR that adds that build step using current best practices I’d be happy to review it and include those wheels going forward.

**Comment by tipabu** [comment]
I think this may have been fixed by #95 -- at any rate, I now see `manylinux` wheels on https://pypi.org/project/xattr/#files

**Comment by etrepum** [comment]
Good catch @tipabu 👍
